versioned_docs/version-V3/guides/providing-liquidity/decrease-liquidity.md

```diff
@@ -0,0 +1,68 @@
+---
+id: decrease-liquidity
+title: Decrease Liquidity
+sidebar_position: 4
+---
+
+Make sure to go through the [first guide](./setting-up-your-contract.md) before continuing to this section
```

Comment:
instead of the 'first guide' lets reference it by name so the user knows what it refers to